### DISCUSSION TITLE
Manage APIs permissions based on verb

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#535] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

I have an API exposed through multiple verbs (GET/POST/PATCH/DELETE).  
I want to give the permission to call the POST/PATCH/DELETE only to the admin, but everyone should be able to call the GET.

How can I do that? In the example below the rule for the GET is applied to `/my-api/`, thus resulting in this situation for a non admin user:
-  `/my-api`: forbidden
- `/my-api/`: access granted

Is there a way to separate the behaviour based on the verb only , whithout introducing a new path?

![image](https://github.com/user-attachments/assets/56145e14-f3e6-4fed-8e42-4ce887a52945)

**Comment by Danielecina** [comment, marked as answer]
Hi @EdoCrippaOfficial, this behavior is not supported. 
Each route needs a path and verb specified to work.

**Reply by Danielecina** [reply]
This can be a useful feature to avoid creating an endpoint like "/" and multiple routes inside them called "GET /my-api", "POST /my-api/", etc...
